docs/source/app-auth-update.rst

```diff
@@ -0,0 +1,42 @@
+Updating an App's credentials
+=============================
+
+Overview
+--------
+
+You may need to change an App's credentials due to internal rotation policies or otherwise. The StreamAlert CLI allows you to easily update App credentials.
+to aid in this process, the CLI also give you the ability to list currently configured Apps.
+
+Listing Apps
+------------
+
+To list currently configured StreamAlert Apps (grouped by cluster), use the CLI command:
+
+.. code-block:: bash
+
+  $ python manage.py app list
+
+This list will be empty for each cluster if no apps are configured, or will appear similar to the following::
```

Comment:
`Example output:`

Reply:
@mime-frame  making this change but then confused myself 🤣  are you looking for example output for what shows when `no apps are configured` or just add this header in general?

Reply:
@mime-frame oh nm I think I got it